packages/main/src/Switch.js

```diff
@@ -275,7 +275,7 @@ class Switch extends UI5Element {
 	}
 
 	get ariaLabelText() {
-		return getEffectiveAriaLabelText(this);
+		return [getEffectiveAriaLabelText(this), this.hiddenText].join(" ");
```

Comment:
Hi, @unazko I'm the reporter of https://github.com/SAP/ui5-webcomponents/issues/4887.
I have some concern about combining the explicit access name specified by the caller of the component and the default internal accessible name.
I think the better way is that if accessible name is specified explicitly, just use it. Otherwise, use the default internal accessible name (`this.hiddenText`).

This is the use case of our application.
We explicitly specify the accessible name of ui-switch by `accessible-nam-ref` (`status-label ui5wc-text-1`). This is evaluated by ui5-switch to `Status Active` and applied to `aria-label` of `div[role="switch"]`. We expect that when this switch get focused, the screen reader announce `Status Active` (rather than `Status Active On`).

Thanks.
<img width="1175" alt="Screen Shot 2022-03-10 at 3 13 41 PM" src="https://user-images.githubusercontent.com/1751846/159392023-97dc16a6-328d-4fea-8c25-c85f0fbe19cb.png">

Reply:
Hello @2bno1,

By accessibility specification there are default readings "on" and "off" that indicate the current checked state of the switch.
Those readings will come out of the box as a result of the "switch" role and aria-checked attribute. We also add
a default "on" and "off" texts explicitly, which isn't needed and by removing them your scenario will work as expected (There won't be "On" or "Off" texts in the aria-label attribute).
For other scenarios however when the <code>text-on</code> and <code>text-off</code> attributes are specified there should be an announcement for those texts. Have in mind that those texts aren't an internal accessible-name, but rather they help indicate the checked state of the switch. For example you could have a switch with "Web server" text for an accessible-name and <code>text-on</code> and <code>text-off</code> texts would be "stop" and "start".
This scenario would produce the following readings - "Web server stop switch off" and "Web server start switch pressed on".

Best Regards,
Boyan

Reply:
@unazko Thanks for your explanation, it makes sense to me. Thank you.